recipes/higher/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "higher" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/higher-{{ version }}.tar.gz
+    sha256: f5b7baa668be5dc59cca9ae06c4ed14c2bd2fe5ca8283f95db660e8732513c1d
+  - path: version.txt
```

Comment:
Recipe looks good to me. I'll merge but could you please maybe open an issue in the upstream repo mentioning that the `LICENSE` and `version.txt` are missing in the sdist?